src/Classes/ServiceAPI/Profile.php

```diff
@@ -57,6 +57,11 @@ public static function getThisYearsMembers()
         return self::getMembersForYear(CoreUtils::getAcademicYear());
     }
 
+    function rem_inx ($str, $ind)
```

Comment:
is this used anywhere?

Reply:
This was when I was attempting to use a separate function to add nicknames to main name column however the table wasn't happy about it and used the method that was previously done to accomplish it. So removed the code that used this